### ISSUE TITLE
Add references to the Foundations chapter

### ISSUE THREAD

**Issue by vcwild** [vcwild/qa4noobs#4]
Add references to the [Foundations](https://github.com/vcwild/qa4noobs/tree/main/docs/00-fundacao) chapter.

A brief example of how it can be done:

e.g.

```md
# Chapter Name

Text

## First topic

Text

### First subtopic

Text
- Bullet point
- Bullet point 2

...

--- Right at the bottom add this ---

## References

- Topic 1
- Topic 2

```

**Comment by vcwild** [comment]
@VictorMPicoli when you create the pull request for this, please mention the following statement:

``` Closes #4```

**Comment by vcwild** [comment]
Closed as won't do
